tutorials/basic-topics/sorting/insertion-sort.md

```diff
@@ -11,4 +11,222 @@ keywords:
   - insertion sort
 ---
 
-<TutorialAuthors names="@TBC"/>
+<TutorialAuthors names="@Shivashish-rwt"/>
+
+## Overview
+
+Insertion sort is one of the sorting algorithms that sort the elements by placing an unsorted element in correct sorted order one at a time.
+
+First of all, let us understand how an sorted `array` (**[Here is a reference for you to check](../arrays.md)**) looks like. Understanding an sorted array will give us the overview of the insertion sort. An sorted array is given below.
```

Comment:
I'm not sure this is Ok to add. It should be basic. 
@wingkwong Any better suggestions?

Reply:
I think we may just mention what sorted array is in a few lines.

Reply:
@wingkwong 
Should i add the what a sorted array is instead of L20.

Reply:
@wingkwong
Is this okay to add:

A Sorted `array` (**[Here is a reference for you to check](../arrays.md)**) is an `array` in which elements are sorted either in ascending or descending order.

Reply:
1. no need to use backtick for `array`
2. you can just write like below then give an example.

``` 
A sorted [array](../arrays.md) is an array in which elements are sorted either in ascending or descending order.
````

Reply:
Done.